components/body/index.jade

```diff
@@ -0,0 +1,55 @@
+.main-content-container.main-side-margin
+  .section-one
+    h1.section-title 純文字連結
+    .section-one-content
+      img(src='./images/Body_1_1.jpg')
+      p Musée du Louvre
+      p Artsy 薈萃全球最頂尖的畫廊、藝術館、基金會、藝術村、藝術博覽會及拍賣會資訊。我們的龐大資料庫不斷更新，現有超過28萬張圖片，搜羅共4萬多位藝術家在藝術、建築及設計範疇的作品，橫跨古代、近代、現代及當代，亦是迄今全球最大的當代藝術網上資料庫。Artsy的用家包括藝術愛好者、博物館常客、藝術資助機構、收藏家、藝術學生及藝術教育工作者，同在Artsy發現藝術、學習藝術、收藏藝術。
+  .section-two
+    h1.section-title Artsy藝術世界
+    .section-two-content
+      .columns
+        .column-one
+          .claude-monet-container
```

Comment:
You might want to scale back here on the specificity. You could name this something like `.artist-container` and `.artist-text` and then you only have two classes to deal with.